### ISSUE TITLE
Handle multiple instances with different API keys more intuitively

### ISSUE THREAD

**Issue by fieldOfView** [fieldOfView/OctoPrintPlugin#46]
The UI currently does not store the API key with the OctoPrint instance, but with the printer instance. It makes more sense to store the API key with the OctoPrint instance, because then it is easier to show which key was entered for which instance.

**Comment by CPS-3** [comment]
When can we expect this issue to be resolved, because i see this issue being removed and added from Cura version to Cura version. Glad someone is known to this issue and will adress it...

**Comment by CPS-3** [comment]
How do you set it to be the same. Can't really figure it out.

**Comment by jacycatlin** [comment]
Yeah, the field is greyed out - I tried forcing it to accept input via Chrome's Inspect feature, but when I enter the new key it doesn't save.

**Comment by fieldOfView** [comment]
Ok, that used to work. I guess now the only way is to edit the OctoPrint configuration file.

I'll put this issue back on top of my list.

**Comment by jacycatlin** [comment]
Hey, editing the .yaml file and rebooting seems to have done the trick!  For now, anyway.  Thanks for helping to shave about 4 steps off my workflow - and here's hoping the final fix arrives someday to get rid of 2-3 more...

**Comment by AkiDK** [comment]
did the change Both printers print at the same time :(

**Comment by fieldOfView** [comment]
This will be fixed in the release for the next version of Cura.

**Comment by jacycatlin** [comment]
Thanks -
